project/Dependencies.scala

```diff
@@ -1,8 +1,8 @@
 import sbt._
 
 object Dependencies {
-  lazy val scalaTest = "org.scalatest" %% "scalatest" % "3.2.7"
-  private val chiselVersion = "3.5-SNAPSHOT"
+  lazy val scalaTest = "org.scalatest" %% "scalatest" % "3.2.14"
+  private val chiselVersion = "3.6-SNAPSHOT"
```

Comment:
Bumping to 3.6-SNAPSHOT is going to mean this library is somewhat unstable. Arguably, this shouldn't have been on 3.5-SNAPSHOT either... Does this work on 3.5.5?

Reply:
Yes, worked nicely on 3.5.5.

Reply:
I will revert to 3.5.5 and force-push.